projects/frontend/shared-components/gui/projects/shared/src/lib/common/predicate/simple/index.ts

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Maybe need to reconfigure `.pre-commit-config.yaml` adding ts support for licensing source file headers?

Reply:
Done.

Reply:
Ticket to update docs https://github.com/vmware/versatile-data-kit/issues/1622